x-pack/qa/full-cluster-restart/build.gradle

```diff
@@ -216,6 +216,8 @@ subprojects {
           setting 'xpack.watcher.encrypt_sensitive_data', 'true'
           keystoreFile 'xpack.watcher.encryption_key', "${mainProject.projectDir}/src/test/resources/system_key"
       }
+      // TODO: Use zen2 in bwc tests
+      setting 'discovery.type', 'zen'
```

Comment:
I _think_ this test should work with Zen2.

Reply:
Same answer as here https://github.com/elastic/elasticsearch/pull/36300#discussion_r239448308.